### DISCUSSION TITLE
Ram did a video about VisiData

### DISCUSSION THREAD

**Discussion by frosencrantz** [saulpw/visidata#1989] [Show and tell]
Looks like @cool-RR made a PyData video about VisiData: https://youtu.be/gdvanybQn8Y `Ram Rachum: Data science in the terminal with VisiData`  

I just found this, but it has been around for a few months.

**Comment by cool-RR** [comment]
@frosencrantz I'm happy you found it. Yep, I posted it on the Discord after I gave the talk. I personally don't follow the Discord, as I imagine many users don't, so I probably would have also missed it if it wasn't my own talk.
